lib/MetaCPAN/Query/Release.pm

```diff
@@ -560,11 +560,28 @@ sub all_by_author {
 }
 
 sub versions {
-    my ( $self, $dist ) = @_;
+    my ( $self, $dist, $versions ) = @_;
 
     my $size = $dist eq 'perl' ? 1000 : 250;
+
+    my $query;
+
+    if ($versions) {
+        $query = {
+            bool => {
+                must => [
+                    { term  => { distribution => $dist } },
+                    { terms => { version      => [ split /,/, $versions ] } },
```

Comment:
It might be better to move this string -> arrayref conversion into the controller.

Reply:
moved